AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1002,8 +1042,8 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertNodes:(NSA
     return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
   }
   
+  [_layoutFacilitator collectionViewWillEditCellsAtIndexPaths:indexPaths batched:_performingBatchUpdates];
   if (_performingBatchUpdates) {
-    [_layoutFacilitator collectionViewWillEditCellsAtIndexPaths:indexPaths batched:YES];
```

Comment:
I think you need to remove the line in the else clause -- it looks like we will call batched:NO twice